src/app/content/html/html-parser/html-parser.service.spec.ts

```diff
@@ -69,13 +82,42 @@ describe('HtmlParserService', () => {
     ['dm:model|dc:class|de:element', '/dataModels/dm:model|dc:class|de:element']
   ])('should rewrite mauro item path %p to valid url %p', (path, pathUrl) => {
     const baseUrl = 'http://localhost/#/catalogue/item';
-    pathNamesStub.createHref.mockImplementationOnce(
+    uiRouterStub.stateService.href.mockImplementationOnce(
       () => `${baseUrl}${pathUrl}`
     );
 
     const content = `<a href="${path}">Mauro item link</a>`;
     const expected = `<a href="${baseUrl}${pathUrl}">Mauro item link</a>`;
-    const modified = service.parseAndModify(content);
+    const modified = service.parseAndModify(content, {});
     expect(modified).toStrictEqual(expected);
   });
+
+  it.each([
```

Comment:
this helps a lot understanding the context of the change. I think this is a good test